user/enterprise/high-availability.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Enterprise High Availability Mode
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise typically runs as a single container instance communicating with one or multiple workers and generally, this setup is quite stable and maintainable. However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode. If you're interested, or might be interested, in HA, please let [us](mailto:enterprise@travis-cicom?subject:HA%2FMode) know and we can discuss options and help you get started. 
+
+## Overview of Installation
+
+The platform installation is quite similar to the standard [Enterprise installation](/user/enterprise/installation#Setting-up-the-Travis-CI-Enterprise-Platform), and the [worker installation is identical](#Setting-up-the-Travis-CI-Enterprise-Worker). However, there are some additional [system prerequisites](/user/enterprise/prerequisites/#High-Availability-Mode), which mean that to install in HA mode, you will need the following:
+ * 3+ VMs with **16 gigs of RAM, 8 CPUs, and 40 GB HDD**, i.e.`c4.2xlarge` with an additional 40GB configured for storage space - this is 2+ VMs for running the Platform, and 1+ for the VMs running the Worker
+ * [Redis](https://redis.io/), [RabbitMQ](https://www.rabbitmq.com/), 
+and [Postgres](https://www.postgresql.org/) instances
+ * [GitHub OAuth app](/user/enterprise/prerequisites#OAuth-App), [trial license](/user/enterprise/prerequisites#License) -- enabled for HA
+ * Internet connection -- note, this installation is _not_ airgapped by default. [Please contact us](mailto:support@travis-ci.com) if you are interested in this. 
```

Comment:
Should this also be enterprise@travis-ci.com?

Reply:
YES 🤦‍♀️